src/components/AccountMenuDropdown.js

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from 'react'
+import Menu from './Menu'
+import ReactCSSTransitionGroup from 'react-addons-css-transition-group'
+
+class AccountMenuDropdown extends Component {
+  constructor(props) {
+    super(props)
```

Comment:
Please add spaces between opening and closing parentheses, and arguments, in a function invocation or control flow expression:
```js
fn(param1, param2) { /* ... */ }
if(expression) { /* ... */ }
```
should be
```js
fn( params1, param2 ) { /* ... */ }
if( expression ) { /* ... */ }
```